tbx/project_styleguide/templates/patterns/pages/service/service_page.html

```diff
@@ -8,8 +8,8 @@
         </div>
 
         {% if page.intro %}
-            <div class="grid__intro streamfield__intro">
-                <div class="heading heading--four">{{page.intro|richtext}}</div>
```

Comment:
Suggestion: Removing the `heading` class has changed the colour and the font weight, I think they should be `white` and `300`. Here's prod vs local:

<details><summary>Details</summary>
<p>

![image](https://github.com/user-attachments/assets/4799429f-cde7-4e6b-951e-3ce67e9ab452)


</p>
</details>

Reply:
This also needs fixing on the Work page

Reply:
that should now be fixed.

Reply:
Looks good!